include/industry_standard/storage.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/**
+ *  Author: Wilfred Mallawa <wilfred.mallawa@wdc.com>
+ *          Alistair Francis <alistair.francis@wdc.com>
+ **/
+
+/** @file
+ * Definitions of SPDM over the Storage as defined in DSP0286
+ **/
+
+#ifndef STORAGE_BINDING_H
+#define STORAGE_BINDING_H
+
+#pragma pack(1)
+
+typedef struct {
+    uint16_t data_length;
+    uint16_t storage_binding_version;
+    uint8_t max_connection_id : 2;
+    uint8_t reserved1[3];
+    uint8_t supported_operations[8];
+    uint8_t reserved2[16];
+} storage_discovery_response_t;
+
+typedef struct {
+    uint16_t data_length;
+    uint16_t storage_binding_version;
+    uint32_t pending_info_flag;
+    uint32_t response_length;
+} storage_pending_info_response_t;
+
+#define STORAGE_SECURITY_BINDING_VERSION 0x1000
```

Comment:
If this is SPDM storage binding spec, please always use `SPDM_STORAGE_` as prefix, as namespace.